charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -109,7 +109,7 @@ spec:
             - "-leader-election"
             - '-handle-volume-inuse-error=false'
             - '-feature-gates=RecoverVolumeExpansionFailure=true'
-            - "-timeout=120s"
+            - "-timeout=240s"
```

Comment:
We'll need to carry these changes over to the latest-v2 chart.

Reply:
Fixed